assets/sass/patterns/organisms/content-aside.scss

```diff
@@ -51,8 +51,140 @@
     }
   }
 
+  .content-aside__second-column {
+    @include padding(24, "bottom");
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+      border-bottom: 1px solid $color-text-dividers;
+    }
+  }
+
+  .js & .content-aside__second-column {
+    display: grid;
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+      display: block;
+    }
+
+    & dt:first-of-type {
+      @include truncate-with-ellipsis();
+      @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+        white-space: unset;
+      }
+    }
+
+    & .definition-list dt,
+    & .definition-list dd {
+      &:nth-of-type(n+2) {
+        display: none;
+      }
+
+      @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+        &:nth-of-type(-n+3) {
+          display: block;
+        }
+        &:nth-of-type(n+5) {
+          display: none;
+        }
+      }
+      @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+        &:nth-of-type(n+2) {
+          display: block;
+        }
+      }
+    }
+
+    .toggle {
+      @include font-size-and-vertical-height(12, 24);
+      @include padding(0 17 0 10);
+      align-self: start;
+      color: $color-text;
+      font-family: $font-secondary;
+      font-weight: bold;
+      grid-row: 1;
+      justify-self: end;
+      position: relative;
+      display: none;
+
+      &:before {
+        content: "History";
+        @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+          content: "Show history";
+        }
+      }
+
+      &:after {
+        @include arrow("down", 5);
+        @include height(5);
+        @include width(5);
+        display: block;
+        position: absolute;
+        right: 0;
+        top: 9px;
+        transform-origin: 50% 50%;
+      }
+
+      @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+        @include padding(0 17 0 0);
+        grid-row: 2;
+        justify-self: start;
+
+        &:after {
+          top: 9px;
+        }
+      }
+
+      @media only all and (max-width: #{get-rem-from-px($bkpt-site--medium - 1)}em) {
+        display: block;
+      }
+
+      // todo: should display on medium viewports in parent has class of toggle--count-gt-3.
+      .toggle--count-gt-3 & {
+        @media only all and (max-width: #{get-rem-from-px($bkpt-site--wide - 1)}em) {
+          display: block;
+        }
```

Comment:
@Cosmin-Daniel @viktoria-mahmud can you help me with this? Should display the toggle trigger on medium viewports if parent with class `toggle--count-gt-3` is present.

Reply:
We can have it above the `&.toggle--expanded .definition-list dt` as `&.toggle--count-gt-3 .toggle`. 
Also, maybe it's better to be inline-block.
![image](https://user-images.githubusercontent.com/107858401/211486712-a3e3a75b-8df3-4073-abd1-a1d90af365e7.png)

Reply:
Thanks @Cosmin-Daniel I have done that and push my commits to the main PR: https://github.com/elifesciences/pattern-library/pull/1212.

I will close this PR now.